package.json

```diff
@@ -56,7 +59,13 @@
     ]
   },
   "eslintConfig": {
-    "extends": "eslint-config-i-am-meticulous/react"
+    "extends": "eslint-config-i-am-meticulous/react",
+    "rules": {
+      "comma-dangle": ["error", {
+        "objects": "ignore",
+        "arrays": "ignore"
+      }]
+    }
```

Comment:
eslint config used is not defining comma-dangle and is only extending eslint:recommended which does not consider this as an error since v3 http://eslint.org/docs/user-guide/migrating-to-3.0.0#changes-to-eslintrecommended

Are you sure it's required? It should not (or I will adjust the lint config...)